### ISSUE TITLE
Victims doesn't show up

### ISSUE THREAD

**Issue by AceFlashX** [AhMyth/AhMyth-Android-RAT#20]
I tried to add a phone over local and dynamic IP / statistic vps IP and it still doesn't show up Victims. I installed the app on different devices and all of them doesn't work.

The port is always the same

**Comment by AhMyth** [comment]
Did you try it locally ?
Did you check your firewall config. ?

**Comment by AceFlashX** [comment]
Yes and yes.
Fuy disabled it.

**Comment by AhMyth** [comment]
which OS are you using ?
Have you tried it on another OS ?

**Comment by AceFlashX** [comment]
Windows 10 and Windows Server 2016. Unfortunately I don't have any other OS.

**Comment by coptenekesi** [comment]
I've tested with windows 8.1, 10 and also windows server 2016. Working very stable actually.

**Comment by mikekok** [comment]
Tried it on both Windows (10) and Linux (Elementary OS), both locally and remotely and the victims never showed up.

**Comment by vincentcox-work** [comment]
I have tried it with the windows release on Windows and the linux release on Kali. It didn't work, so I build it from source. Which also didn't worked.

**Comment by svargas28** [comment]
please help me...

**Comment by mikekok** [comment]
@svargas28 did you try building it from source as well? It worked, at least for me.

**Comment by svargas28** [comment]
@mikekok I will try, thank you.

**Comment by svargas28** [comment]
@mikekok I generate the apk from the source, but on the android it says that it can not be installed.

**Comment by mikekok** [comment]
Well, I did not have that problem at all. Have you enabled the ability to install apps from unknown sources on your android device?

**Comment by svargas28** [comment]
yes, it is already installed on the android, but the PC is not shown

**Comment by mayday77** [comment]
Is any options to comunicate with you by email / telegram ?

**Comment by BOSS007a47** [comment]
victims does not pop up.Can It be that I'm using my phone for internet access?Please let me know.

**Comment by os-prashant** [comment]
i am open ahmyth using terminal > sudo ahmyth
this issue showing on terminal
i am using Ubuntu 19.04
my firewall is disabled 

(ahmyth:2093): GConf-WARNING **: 11:56:11.117: Client failed to connect to the D-BUS daemon:
Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.

**Comment by Pyanne** [comment]
Same issue for me.
I followed @os-prashant video to install Ahmyth on Kali in a Virtualbox. I made another virtualbox with an Android-x86. Both are on local network with 192.168.0.* adresses.
Ahmyth is listening but sees nothing.

**Comment by Untamedwolf** [comment]
Ahmyth does not open after installation! (Os Kali)Anyone else facing the same issue?
Any fix to it?

**Comment by chandusingh512** [comment]
Listening on port => 42474 is showing but not showing victims

**Comment by HashTrak** [comment]
same here @chandusingh512  bro

**Comment by IGARMAAN** [comment]
I am using AhMyth application and after successfully creating a payload, I installed the payload in my android device. I entered port number and clicked on Listen button, my device is not showing in VictimsLab. What do I do now?? Please help me if you know the solution.

**Comment by chandusingh512** [comment]
> same here @chandusingh512  bro
> 

I used fatrat and that is working for me

**Comment by chandusingh512** [comment]
> I am using AhMyth application and after successfully creating a payload, I installed the payload in my android device. I entered port number and clicked on Listen button, my device is not showing in VictimsLab. What do I do now?? Please help me if you know the solution.

Using Fatrat u can listen through msfconsole
